### PR TITLE
Standardize readme shield badges

### PR DESCRIPTION
**Pull request by MO2k4**
# Description

Standardize readme shield badges

## Issue reference

Please reference the issue this PR will close: #7738 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [x] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,32 @@
...
-[![Build Status](https://github.com/dapr/dapr/workflows/dapr/badge.svg?event=push&branch=master)](https://github.com/dapr/dapr/actions?workflow=dapr)
-[![E2E Tests](https://img.shields.io/endpoint?url=https://gist.githubusercontent.com/dapr-bot/14e974e8fd6c6eab03a2475beb1d547a/raw/dapr-test-badge.json)](https://github.com/dapr/dapr/actions?workflow=dapr-test&event=schedule)
-[![codecov](https://codecov.io/gh/dapr/dapr/branch/master/graph/badge.svg)](https://codecov.io/gh/dapr/dapr)
-[![Discord](https://img.shields.io/discord/778680217417809931)](https://discord.com/channels/778680217417809931/778680217417809934)
-[![License: Apache 2.0](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/dapr/dapr/blob/master/LICENSE)
-[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fdapr%2Fdapr.svg?type=shield)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fdapr%2Fdapr?ref=badge_shield)
-[![TODOs](https://badgen.net/https/api.tickgit.com/badgen/github.com/dapr/dapr)](https://www.tickgit.com/browse?repo=github.com/dapr/dapr)
```
[the remark this answers is not in the document]
Comment (MO2k4) [reply README.md:14]: Do you mean that the todo badge is missing and should be added?

[the remark this answers is not in the document]
Comment (daixiang0) [reply README.md:14]: Yep.

[the remark this answers is not in the document]
Comment (MO2k4) [reply README.md:14]: its added :)

**Review by yaron2 — APPROVED** [review]

**Comment by marcduiker** [comment]
@holopin-bot @MO2k4 Thank you, enjoy your Dapr badge!

**Pull request merged**